abi/type_test.go

```diff
@@ -505,6 +601,17 @@ func TestTypeMISC(t *testing.T) {
 	for _, testcaseTuple := range testpoolTuple {
 		require.True(t, testcaseTuple.Equal(testcaseTuple), "test type tuple equal error")
 	}
+	// add testcase for array of tuples into testpool
+	testpoolTupleArray := make([]Type, 0, 200)
+	for i := 0; i < 100; i++ {
+		testpoolTupleArray = append(testpoolTupleArray, makeDynamicArrayType(testpoolTuple[i]))
+	}
+	for i := 0; i < 100; i++ {
+		testpoolTupleArray = append(testpoolTupleArray, makeStaticArrayType(testpoolTuple[i], 10))
+	}
+	for i := 0; i < 100; i++ {
```

Comment:
should this be 200?

Reply:
this can be any number: I am here sampling 100 tuple types from
- types in `testpool`
- types in `testpoolTupleArray` (which has length 200)

Reply:
oh I see, it just gets appended below.